tests/integration/query/simple/with_filter/with_alias_test.go

```diff
@@ -0,0 +1,322 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package simple
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestQuerySimple_WithAliasEqualsFilterBlock_Succeeds(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple query with alias filter(age)",
+		Actions: []any{
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "John",
+					"Age": 21
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Bob",
+					"Age": 32
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+					Users(filter: {_alias: {UserAge: {_eq: 21}}}) {
+						Name
+						UserAge: Age
+					}
+				}`,
+				Results: map[string]any{
+					"Users": []map[string]any{
+						{
+							"Name":    "John",
+							"UserAge": int64(21),
+						},
+					},
+				},
+			},
+		},
+	}
+
+	executeTestCase(t, test)
+}
+
+func TestQuerySimple_WithEmptyAlias_Succeeds(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple query with empty alias filter",
+		Actions: []any{
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "John",
+					"Age": 21
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Bob",
+					"Age": 32
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+					Users(filter: {_alias: {}}) {
```

Comment:
question: Is this consistent with the behaviour of other 'empty' filter properties?

Reply:
Yes, there's no existing tests but I double checked

Reply:
Nice, thanks Keenan :)